### PR TITLE
Implements image tags

### PR DESCRIPTION
**Pull request by rishabsjain**
Implements image tags using acts_as_taggable gem

Closes Issue #6 
updates views to include tags in an appropriate way on each page
Implements acts_as_taggable gem into image model and creates needed db migrations

### CODE REVIEW

File: app/assets/stylesheets/show.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+.img {
+    display: inline-block; max-width:500px;
+}
+.img img {
```
Comment (zjswanson) [inline app/assets/stylesheets/show.css:5]: I think this style isn't actually doing anything.  This selector says "Find elements with the class `img` and then find any `` child elements and apply these styles.  This wouldn't apply to your images in the current markup.

File: app/views/image/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,7 @@
-<p> Image <%=@image.id%> </p>
-<img src=<%=@image.url%> >
\ No newline at end of file
+<style>
```
Comment (zjswanson) [inline app/views/image/show.html.erb:1]: It's not a good idea to add styles directly in the template unless absolutely necessary.  They will override styles from your stylesheets and be harder to find down the road.  In this case, you could accomplish the same result with the bootstrap `text-center` class on these elements.

File: test/models/image_test.rb [hunk 3]
```diff
@@ -5,21 +5,60 @@ class ImageTest < ActiveSupport::TestCase
   #   assert true
   # end
   def test_correctly_formatted_urls_valid
-    temp = Image.new(url: 'http://url.com/image.jpg')
+    temp = Image.new(
```
Comment (zjswanson) [inline test/models/image_test.rb:8]: I might update the test names here as it's not just testing the url format anymore.  perhaps `create_image_with_valid_params` or something like that

**Review by zjswanson — CHANGES_REQUESTED** [review]

File: test/models/image_test.rb (hunk 3 above)
Comment (rishabsjain) [reply test/models/image_test.rb:8]: I need the test at the front, so I tried to make it more descriptive

**Review by zjswanson — APPROVED** [review]
🌮

**Pull request merged**